runner/ansible/README.md

```diff
@@ -22,3 +41,84 @@ These are the fields needed by Trento:
 is that the labels are used for control purpose (select all the checks with this label e.g.), and the groups are used just for visual purposes
 - `description`: A longer description about the test purpose. It can be written using markdown.
 - `implementation`: Usually the task `main.yml` content
+
+## Check files
+
+The check files contain the actual task which the runner will execute. An
+example is available at [tasks/main.yml](roles/checks/1.1.1/tasks/main.yml).
+These files are written like a normal ansible task, but:
+
+- The tasks are of `read only` nature, meaning that they are meant to check for things
```

Comment:
This 1st bullet list entry looks confusing. I would rewrite it as:
`The checks are of `read-only` nature, so they must not apply any change on the system.